### ISSUE TITLE
Hamburger menu is duplicated

### ISSUE THREAD

**Issue by nicolae-stroncea** [ActivityWatch/aw-android#45]
Currently the app has 2 hamburger menus (one from web-ui, the other one specifically for Android) which leads to very confusing UX.

I'm not sure what the purpose of the Android menu is, since the aw-webui menu already contains all of the information, so an easy fix might be to just remove the Android one.

Any UI customizations that the android menu has, can be done directly on the web-ui with a media query that only executes for certain screen widths(i.e mobile).

```javascript
@media only screen and (max-width: 768px) {

    #hamburger-menu {
        position: sticky;
        top: 0;
    }

}

```

**Comment by ErikBjare** [comment]
The only reason for the native Android menu is to get a slightly more native feel. The idea was to eventually hide the menu in aw-webui when running on Android. 

It's still duplicating code though, so idk.

**Comment by ErikBjare** [comment]
This was fixed in v0.10, closing.
